### DISCUSSION TITLE
Issues installing foundry on windows 11

### DISCUSSION THREAD

**Discussion by Ehkayy** [Cyfrin/foundry-full-course-cu#3110] [Q&A]
"I am receiving an error when using VS code terminal to paste the bash." :

Invoke-WebRequest : A parameter cannot be found that matches parameter name 'L'.
At line:1 char:6
+ curl -L https://foundry.paradigm.xyz | bash
+      ~~
    + CategoryInfo          : InvalidArgument: (:) [Invoke-WebRequest], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
    
    Here is my code:
    PS C:\FrontEnd\foundry> curl -L https://foundry.paradigm.xyz | bash
    
    Could someone please help me figure out what the issue is?

**Comment by EngrPips** [comment]
Hello @Ehkayy, Please read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2518#discussioncomment-10713419 I believe it will help you.

**Reply by Ehkayy** [reply]
okay thank you

**Reply by EngrPips** [reply]
You are welcome.

**Comment by Ehkayy** [comment]
Hello EngrPips thank you for your help with the article but the guide is
stated in the article to be for windows 10
I am using Windows 11 pro, should I use the same process meant for windows
10 from the article?

**Reply by EngrPips** [reply]
I will say give it a try and let's see if works.

**Comment by Ehkayy** [comment]
It's alright
Thank you
